skimage/morphology/convex_hull.py

```diff
@@ -93,40 +93,66 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
     return mask
 
 
-def convex_hull_object(image, neighbors=8):
-    """Compute the convex hull image of individual objects in a binary image.
+def convex_hull_object(image, neighbors=None, connectivity=None):
+    r"""Compute the convex hull image of individual objects in a binary image.
 
     The convex hull is the set of pixels included in the smallest convex
     polygon that surround all white pixels in the input image.
 
     Parameters
     ----------
-    image : (M, N) array
+    image : (M, N) ndarray
         Binary input image.
     neighbors : {4, 8}, int
-        Whether to use 4- or 8-connectivity.
+        Whether to use 4 or 8 adjacent pixels as neighbors.
+        If ``None``, set to 8. **Deprecated, use** ``connectivity`` **instead.**
+    connectivity : {1, 2}, int
+        Determines the neighbors of each pixel. Adjacent elements
+        within a squared distance of ``connectivity`` from pixel center
+        are considered neighbors. If ``None``, set to 2::
```

Comment:
I guess that in 0.18, we need another deprecation cycle to set this option to 2 (A note in TODO?).